shopfloor/views/product_packaging_type.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_product_packaging_type_form" model="ir.ui.view">
+        <field name="name">shopfloor.product.packaging.type.form</field>
+        <field name="model">product.packaging.type</field>
+        <field
+            name="inherit_id"
+            ref="product_packaging_type.view_product_packaging_type_form"
+        />
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <div>
```

Comment:
Is the enclosing `<div>` needed ?

Reply:
This is to move this field to the next line (if I don't add it, the image appears on the same row as the name of the packaging in the form).